### ISSUE TITLE
[Language server] Signature Help feature doesn't work with "(" character

### ISSUE THREAD

**Issue by SkorikSergey** [eclipse-che/che#10699]
**Reproduction Steps:**
1. Create and start workspace from **PHP** stack with "web-php-simple" project.
2. Open **index.php** file and change text to:
```
<?php
function sum($x, $y) {
    return $x + $y;
}
echo "14 + 16 = " . sum(14,16)
?>
```
3. Create a new line in position 6 and type ```sum(```.

**Expected behavior:**
Hover with ```(mixed $x, mixed $y)``` message appeared.

**Observed behavior:**
Nothing changes. The feature will work only with "," character.
 
**Che version**: 6.10.0
**OS and version:** Fedora 28
**Language servers that supported this feature:** PHP, Python, Clang, Type Script, Golang, C#.

**Additional information:**
* Problem can be reliably reproduced, doesn't happen randomly: [Yes]
* See the attachment:

**Screencast:**
![screencast-che-eclipse-che 172 19 20 234 nip io-2018 08 08-13-11-02](https://user-images.githubusercontent.com/7760565/43831369-93e65738-9b0c-11e8-9dee-65d13889c746.gif)

**Screenshot:**
Hover will appear only after "," character.
![selection_071](https://user-images.githubusercontent.com/7760565/43831428-c4464cc6-9b0c-11e8-8c98-5ad4ef43b3b6.png)

**Comment by tsmaeder** [comment]
There is no code in the signature help provider that hard codes any particular trigger character. Most likely, the php language server does not provide "(" as a trigger character. That be a problem with the PHP language server outside of our control.

**Comment by SkorikSergey** [comment]
![selection_126](https://user-images.githubusercontent.com/7760565/45555356-4718e900-b841-11e8-9a78-48c8e1e3690c.png)

**Comment by musienko-maxim** [comment]
So as we've just clarified this feature works as well with Ctrl+P key.  And we should implement this in our tests. But this one also is invoked by `((` and `,` symbols. And this may be bring potential conflicts. We have to think how it should work together. 
Also after invocation of `(` symbol we have unexpected exceptions in the dev-machine console under `TypeScript` LS. So, we are going to check this under other language servers and report about this in this issue.
